text/0000-non-ascii-idents.md

```diff
@@ -142,6 +142,17 @@ These are the three different naming conventions and how their corresponding lin
 
 Note: Scripts with upper- and lowercase variants ("bicameral scripts") behave similar to ASCII. Scripts without this distinction ("unicameral scripts") are also usable but all identifiers look the same regardless if they refer to a type, variable or constant. Underscores can be used to separate words in unicameral scripts even in UpperCamelCase contexts.
 
+## Reusability
+
+The code used for implementing the various lints and checks will be released to crates.io. This includes:
+
+ - Testing validity of an identifier
+ - Testing for `less_used_codepoints` ([UTS #39 Section 3.1][TR39Allowed])
+ - Script identification and comparison for `mixed_script_detection`  ([UTS #39 Section 5.2][TR39RestrictionLevel])
+ - `skeleton(X)` algorithm for confusable detection ([UTS #39 Section 4][TR39Confusable])
+
```

Comment:
Could you add a note here along the lines of "These checks can serve a similar purpose when comparing text in Rust code to text supplied by a user, such as when debugging URL routing or command-line option parsing."

Reply:
Not sure why this isn't covered by the subsequent paragraph, added an explicit mention of user supplied strings.